File: PlatziCodingChallenge_Dia4/RepiteLaPalabra.py
#Repite la palabra en Python
def repeat(word, times):
    if times <= 1:
        return word
    else:
        return word + repeat(word, times - 1)

palabra = input('¿Qué palabra quieres repetir? ')
veces = int(input('¿Cuántas veces quieres repetirla? '))
print(repeat(palabra, veces))

File: Dia04/RepiteLaPalabra.js
//Repite la palabra en JS
function repeat(word, times) {
    if(times <= 1){
        return word
    } else {
        return word + repeat(word, times - 1)
    }
}

var palabra = prompt('¿Qué palabra quiere repetir?')
var veces = parseInt(prompt('¿Cuántas veces quiere repetirla?'), 10)
alert(repeat(palabra, veces))